scripts/release.sh

```diff
@@ -0,0 +1,307 @@
+#!/bin/bash
+#
+# Name::        release.sh
+# Description:: Use this script to prepare a new release on Github,
+#               the automation will create a GH tag like 'v0.1.0'
+#               (using the VERSION file)
+# Author::      Salim Afiune Maya (<afiune@lacework.net>)
+#
+set -eou pipefail
+
+readonly org_name=lacework
+readonly project_name=terraform-kubernetes-agent
```

Comment:
this is really important:
```suggestion
readonly project_name=terraform-aws-ssm-agent
```